pkg/roles/roles.go

```diff
@@ -15,9 +16,25 @@ const (
 	Build       = "acorn:project:build"
 	ClusterView = "acorn:cluster:view"
 	ClusterEdit = "acorn:cluster:edit"
+
+	// AWS
+	GroupAWSAcornIO     = "aws.acorn.io"
+	GroupRoleAWSAcornIO = "role.aws.acorn.io"
+
+	AWSAdmin = "acorn:aws:admin"
 )
 
 var (
+	awsRoles = map[string][]rbacv1.PolicyRule{
+		AWSAdmin: {
+			{
+				APIGroups: []string{awspermissions.AWSAPIGroup, awspermissions.AWSRoleAPIGroup},
+				Verbs:     []string{"*"},
+				Resources: []string{"*"},
+			},
+		},
+	}
```

Comment:
Do we have a practical use case/need for this? This seems really scary to me.

Reply:
Yeah, basically for all allowed aws service acorns.
The role will be referenced by IRAs in Runtime and IRBs in Manager.
We'll have the almighty admin role until we figure out more granular roles that make sense for the services.
So far we had auto-approvals in Manager.

Reply:
I'm sure that this has already been discussed, but do these roles need to live in Runtime? They feel more Manager oriented to me.